frontend/src/app/components/crm/guide/guide.component.html

```diff
@@ -1 +1,25 @@
-<p>guide works!</p>
+<div id="content" fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="57px">
+    <div id="settingUp" class="guideModule" fxLayout="row" fxLayoutAlign="space-between start" fxLayoutGap="100px">
+        <div id="settingUpText">
+            <span class="mat-h1">
+                Setting Up:
+            </span>
+            <ol>
+                <li>At the top right of the page, click on the User icon.</li>
+                <li>Your unique webhook and google key should appear, copy them to the Google Form Ad(s) that you wish to link to miniCRM.
```

Comment:
Capitalize `google` in `google key` and change `webhook` to `webhook url`.

Reply:
fixed